---
title: Research in the Pathogen Genomics and Evolution Group
layout: default
group: research
---

<div class="row">

# Our research
As a group, our fundamental interest is in bacterial evolution, particularly the emergence and transmission of bacterial pathogens. We are interested in how and when novel clones emerge, how they become resistant to antibiotics, how they interact with human and animal hosts, and what are their pathways of transmission. We predominantly use large-scale genomics, phylogenetics and population genomics to address these questions. As we are interested in fundamental processes, we do not limit ourselves to specific organisms, but try to look for similar processes operating in different organisms and different niches. The work of the group covers a diverse range of topics within this broad area:

</div>

<div class="row">

<h5>1) Development of analytical approaches </h5>

We have developed tools to describe and understand the pangenome (e.g. [**Panaroo**](https://gtonkinhill.github.io/panaroo/#/)), and we are developing approaches to model and incorporate non-SNP variation in phylogenetic inference. We are very interested in applying mutational signatures to understanding bacterial niches, and have developed software to enable this. In the past we have developed novel GWAS methods, and have a continued interest in this area.

<h5>2) Application of phylogenetics </h5>

We use phylogenetic approaches to understand the emergence and evolution of bacterial clones, and the adaption of these clones to novel niches. These include lung pathogens affecting people with CF, such as *Mycobacterium abscessus* and *Pseudomonas aeruginosa*.

We use the same tools to study the transmission pathways of other human and animal pathogens, including *Mycobacterium bovis*, *Campylobacter jejuni*, *Bordetella pertussis*, *Salmonella enterica*, *Escherichia coli*, *Streptococcus equi* and many others. These studies overlap strongly with the use of genomics in clinical microbiology, where we have a long-standing collaboration with [**Prof. Sharon Peacock**](https://www.med.cam.ac.uk/peacock/), investigating the implementation of transmission tracking for *Staphylococcus aureus*, amongst others.

We also use these phylogenies to study the temporal patterns of emergence and transmission, and to uncover phenotype-genotype associations using GWAS.

<h5> 3) Metagenomics </h5>

We have been involved in many metagenomics studies of the gut and the lung microbiota, and have a particular interest in low-biomass studies and the thorny issues of false-positive signals due to pervasive contamination.

<h5> 4) Experimental studies </h5>

We use high-throughput mutagenesis approaches (such as TraDIS, previously developed in our lab) and other â€˜omics tools such as dual-RNA Seq, to study the interaction of pathogens with the host. This currently includes work on *Streptococcus zooepidemicus* and *Klebsiella pneumoniae*. We use similar approaches to study the mode of action of antimicrobials and the development of antibiotic resistance.

</div>

<div class="row">

<h4> Our collaborators </h4>

We have a number of ongoing collaborations with leading researchers around the globe including, but not limited to the following:

<style>
table {
  font-family: arial, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #dddddd;
  text-align: left;
  padding: 8px;
}

tr:nth-child(even) {
  background-color: #dddddd;
}
</style>

<table>
  <tr>
    <th>Collaborator</th>
    <th>Affiliation</th>
  </tr>
  <tr>
    <td>[**Prof. Sharon Peacock**](https://www.med.cam.ac.uk/peacock/)</td>
    <td>Department of Medicine, University of Cambridge</td>
  </tr>
</table>

</div>